src/components/Footer.js

```diff
@@ -1,7 +1,24 @@
-import React from "react";
+import * as React from "react";
+import Box from "@mui/material/Box";
+import BottomNavigation from "@mui/material/BottomNavigation";
+import BottomNavigationAction from "@mui/material/BottomNavigationAction";
 
-function Footer() {
-  return <div></div>;
-}
+export default function SimpleBottomNavigation() {
+  const [value, setValue] = React.useState(0);
```

Comment:
I don't know Material UI that well.  What's the reasoning behind initializing the `value` here to `0` instead of something like `null`?

Reply:
Tested in sandbox and `null` works too when I initialize with ti